packages/www/pages/docs/livepeer-api/webhooks.mdx

```diff
@@ -0,0 +1,105 @@
+import DocsLayout from "../../../components/DocsLayout";
+
+<DocsLayout>
+
+## Wehbooks
+
+**`NOTE: This is still in development`**
+
+Livepeer offers a standard webhook to integrate with your platform's backend. Currently the following events are supported:
+
+- `streamStarted` : triggered when a stream is started.
```

Comment:
We should explain what this event does - something like "this event will be triggered whenever a new stream is started associated with your account".